pkg/omf/functions/bundle/omf.bundle.add.fish

```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
```

Comment:
`-l` was not used in `omf.bundle.remove.fish`. Would you like the `-l` flag there as well?